core/variant/variant_construct.cpp

```diff
@@ -211,6 +211,8 @@ void Variant::_register_variant_constructors() {
 	add_constructor<VariantConstructorToArray<PackedVector2Array>>(sarray("from"));
 	add_constructor<VariantConstructorToArray<PackedVector3Array>>(sarray("from"));
 	add_constructor<VariantConstructorToArray<PackedColorArray>>(sarray("from"));
+	add_constructor<VariantConstructorToNullFilledArray>(sarray("initial_size"));
+	add_constructor<VariantConstructorToValueFilledArray>(sarray("initial_size", "value"));
```

Comment:
Instead of adding the constructors, we can bind the static method (`Array.create_filled_array()`). Moreover, I think we shouldn't add both a constructor and a static method that do the same thing.

I'm not sure which option is better, but for several times we have preferred a static method. See #57477.

Reply:
I think it makes more sense as a static method, both in Array and Variant. Done.